packages/owl-bot/src/fetch-configs.ts

```diff
@@ -41,6 +41,6 @@ export async function fetchConfigs(
     return collectConfigs(repoDir);
   } finally {
     // When running in cloud functions, space is limited, so clean up.
-    fs.rmdirSync(tmpDir, {recursive: true});
+    fs.rmSync(tmpDir, {recursive: true});
```

Comment:
one slight behaviorial change is that this will now throw if the path doesn't exist, this is probably fine, if there's a reason why we expect the folder not to exist we can add the `force` flag.